xknx/devices/sensor.py

```diff
@@ -40,6 +375,14 @@ def __init__(
     ):
         """Initialize Sensor class."""
         super().__init__(xknx, name, device_updated_cb)
+
+        if value_type is not None and not self.validate_value_type(value_type):
+            raise DeviceIllegalValue(
+                value_type,
+                f"value_type '{value_type}' not supported. Supported types are: "
+                f"{[dpt.value_type for dpt in self.supported_dpts]}",
+            )
+
```

Comment:
This is already checked https://github.com/XKNX/xknx/blob/16a10230aa6cf86e98fcafc7460163471f0bf730/xknx/remote_value/remote_value_sensor.py#L42-L48
I don't think there is a need to import all DPT subclasses for this here.

An xknx Sensor class can be used for any DPT - it doesn't have the constraints of a HA SensorEntity (numeric, string, datetime).
I'd like to generally avoid having HA specific things in xknx.

Reply:
Got it. I initially assumed the restriction was on the library side, which is why I included it here.